Octokit.Tests/Clients/GistsClientTests.cs

```diff
@@ -1,321 +1,573 @@
 ﻿using NSubstitute;
-using Octokit;
 using Octokit.Internal;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Threading.Tasks;
 using Xunit;
 
-public class GistsClientTests
+namespace Octokit.Tests.Clients
```

Comment:
We'll circle around to that later and get R# to play along (I'm pretty sure that's R#, it is on my machine at least)